Library/Homebrew/cask/auditor.rb

```diff
@@ -48,12 +48,22 @@ def initialize(
       @except = except
     end
 
+    LANGUAGE_BLOCK_LIMIT = 10
+
     def audit
       warnings = Set.new
       errors = Set.new
 
       if !language && language_blocks
-        language_blocks.each_key do |l|
+        if language_blocks.length > LANGUAGE_BLOCK_LIMIT && !@audit_new_cask
+          sample_keys = language_blocks.keys.sample(LANGUAGE_BLOCK_LIMIT)
+          sample_languages = language_blocks.select { |k| sample_keys.include?(k) }
+          ohai "Auditing a sample of available languages: #{sample_keys.map { |lang| lang[0].to_s }.to_sentence}"
```

Comment:
```suggestion
          ohai "Auditing a sample of available languages: #{sample_keys.to_sentence}"
```

I feel like the above should work, however the last item is printed with `[" "]` syntax.

```
==> Auditing a sample of available languages: es-AR, ur, cs, th, sv, ca, pl, es-CL, he and ["es-ES"]
```

Reply:
@MikeMcQuaid I am converting to keys to enable the use of the `#sample` method that is only available for arrays, and then filtering the languages for the ones that match the sample.